packages/components/src/custom-select-control/style.scss

```diff
@@ -41,8 +41,12 @@
 		height: 100%;
 		padding: 0;
 		position: absolute;
-		right: 0;
+		right: $grid-unit-10;
 		top: 0;
+
+		&:not(.is-next-36px-default-size) {
```

Comment:
The :not selector adds a lot of specificity. But that's probably fine here? This is for the component that is to be deprecated in favor of the default 36px size, correct?

Reply:
Correct 👍 And the only reason we're structuring these as the negative `:not(.is-next)` instead of `&.is-next` is so we can keep the deprecated code grouped together for easier deletion in the future.